src/PrestaShopBundle/Translation/Provider/AbstractProvider.php

```diff
@@ -116,23 +115,6 @@ public function setDomain($domain)
         return $this;
     }
 
-    /**
-     * Get the PrestaShop locale from real locale.
-     *
-     * @return string The PrestaShop locale
-     *
-     * @deprecated since 1.7.6, to be removed in the next major
-     */
-    public function getPrestaShopLocale()
```

Comment:
We tried to remove all deprecated functions that were left from 1.4, 1.5 and 1.6

Removing 1.7 functions is a little bit more sensible because it makes module developers work. But I acknowledge not all functions have the same impact.

@kpodemski I think removing this function is safe, I'm not sure a lot of module rely on a Translation Provider to get the locale?